feat: add solana recipe (#208)
reown-com/reown-docs: docs/appkit/recipes/solana-send-transaction.mdx

---

> title: Send a Transaction in AppKit Web with Solana

Reviewer: ```suggestion
title: Send a Solana Transaction Using AppKit Web
```

---

import GithubBox from '../../components/GithubBox'

# How to Sign Messages, Send Transactions, and Get Balance in Solana using AppKit

Learn how to use Reown AppKit for essential wallet functionalities such as signing messages, sending transactions, and retrieving wallet balances.

---

In this recipe, you will learn how to:

- Retrieve the balance of the connected wallet. 
- Sign a message using a connected wallet.

> - Send a transaction to a Solana blockchain.

Reviewer: ```suggestion
- Send a transaction to the Solana blockchain.
```

Grammar fix

This guide takes approximately 20 minutes to complete.

Let’s dive in!

![](/assets/solanaGuide.gif)

## Prerequisites
- A fundamental understanding of JavaScript and React.
- A minimal installation of AppKit in React.
- Obtain a new project Id on Reown Cloud at https://cloud.reown.com

## Final project

> <GithubBox name="Appkit Solana Example with blockchain interactions" url="https://github.com/reown-com/appkit-web-examples/tree/main/react/react-solana" description="Download the full project to try it directly on your computer."></GithubBox>

Reviewer: ```suggestion
<GithubBox name="Appkit Example for Interacting with the Solana Blockchain" url="https://github.com/reown-com/appkit-web-examples/tree/main/react/react-solana" description="Clone this Github repo to try it out locally."></GithubBox>
```

Grammar fix
